astro/src/content/docs/apis/_application-search-request-parameters.mdx

```diff
@@ -5,7 +5,7 @@ import InlineField from 'src/components/InlineField.astro';
 {/*  parameter_prefix is either blank for parameters or "search." for body */}
 
 <APIBlock>
-  <APIField name={props.parameter_prefix + 'expand'} type="Array<String>" optional defaults="[roles, scopes]" since="1.50.0">
+  <APIField name='expand' type="Array<String>" optional defaults="[roles, scopes]" since="1.50.0">
```

Comment:
```suggestion
  <APIField name="expand" type="Array<String>" optional defaults="[roles, scopes]" since="1.50.0">
```

For consistency, prefer double quotes.